kn/base/templatetags/base.py

```diff
@@ -25,6 +26,15 @@ def email_filter(value):
 def mark_safe_filter(value):
     return mark_safe(value)
 
+@stringfilter
+@register.filter(name='json')
+def json_filter(data):
+    return mark_safe(json.dumps(data)
+                         .replace('&', '\u0026')
+                         .replace('<', '\u003c')
+                         .replace('>', '\u003e')
+                         .replace('/', '\u002f'))
```

Comment:
Ik heb wel een suggestie  voor een ietwat lelijke workaround die zeker veilig is, mocht je nog niet zeker zijn over je eigen.  Zet het JSON object om naar een string.  Doe dan een base64 encoding over deze string.  Deze string is makkelijke veilig in javascript te krijgen.  Dan kun je in javascript de base64 decoden en de json parsen.